### ISSUE TITLE
GV starting working again for me yesterday...

### ISSUE THREAD

**Issue by dnasralla** [d235j/GVFixer#7]
I decided to run Growl Voice yesterday on a whim and it booted up fine. Is it working for others? I am using the GVFixer.

**Comment by Critter** [comment]
aye. same here. Several others have mentioned it on twitter as well...

**Comment by mogilner** [comment]
It is working for me also. I am also able to use it in OS X El Capitan.

**Comment by Fofer** [comment]
Yes!  Working here too!  Thanks so much for posting, dnasralla!  This has made my day!

(Also nice to hear that EasySIMBL and GVFixer are working on OS X El Capitan, so thanks for mentioning that too, mogliner.  I've read that other SIMBL plugins like SafariStand will need updating.)

**Comment by dnasralla** [comment]
Great! Glad to hear other good reports. I was afraid to reboot my computer in case it might  failed!

**Comment by Fofer** [comment]
I'm happy to have it back, and now, more aware than ever, we're on borrowed time and it may very well break again sometime soon. For good.

In the meantime, can we somehow Retina-ize the menulet?  It stands out like an ugly, blurry sore thumb in my menu bar.  ;)

**Comment by ScootRay** [comment]
It's working for me as well, yaaaay!

**Comment by Fofer** [comment]
And... it stopped working for me again. 

"Error: Unable to Sign In...
The operation couldn't be completed.
(com.google.HTTPStatus error 404.)"

Anyone else?  Is it dead... again?

**Comment by Critter** [comment]
@Fofer it's working for me, but I'm only on 10.10.4 and doing my best not to logout for any reason

**Comment by d235j** [comment]
Yep, it's dead again. Once you logout you won't be able to log back in.

**Comment by Fofer** [comment]
Damn.  It was good while it lasted, I suppose. Again.

/shakes fist at Google
